cdn-images-api/main.tf

```diff
@@ -0,0 +1,55 @@
+resource "aws_cloudfront_distribution" "images_cdn" {
+  optimizer {
+    origin {
+      domain_name = "dev-k8s.treetracker.org"
```

Comment:
Since this is for dev, will you also make a PR in the future for test and prod?

Reply:
Yes I will. First I will deploy in the dev env to check that everything works well, after that I will do same in the test and pro env.